docs/aqt/devices.md

```diff
@@ -0,0 +1,23 @@
+# AQT devices
+
+AQT offers access to several quantum computing devices, so-called backends, ranging from real-hardware ion traps with various number of ions to quantum computing simulators including different noise models. To get an overview of available devices visit [www.aqt.eu](https://www.aqt.eu) and get direct access to the devices via the [AQT gateway portal](https://api-gateway.aqt.eu).
+
+## Tokens
+
+The AQT API to access backends uses token-based authentication. In order to be able to submit quantum circuits via quantum programming software development kits you need to supply these tokens. Once you have successfully subscribed to an AQT backend you can retrieve the token on the [AQT gateway portal](https://api-gateway.aqt.eu) and use it in your quantum programs or Jupyter notebook tutorials.
```

Comment:
same here - gateway-portal.aqt.eu and gateway.aqt.eu seem to work, api-gateway.aqt.eu doesn't

Reply:
thanks for pointing that out, updated URLs accordingly